fla/ops/mala/naive.py

```
@@ -0,0 +1,82 @@
+        k (torch.Tensor): Keys of shape [B, T, H, K].
+        v (torch.Tensor): Values of shape [B, T, H, V].
+        sin (torch.Tensor, optional): Sinusoidal position embeddings of shape [T, K].
+        cos (torch.Tensor, optional): Cosine position embeddings of shape [T, K].
+        scale (float, optional): Scale factor for attention scores.
+        
+    Returns:
+        torch.Tensor: Output of shape [B, T, H, V].
+    """
+    dtype = q.dtype
+    q, k, v = map(lambda x: x.float(), (q, k, v))
+    B, T, H, K, V = *q.shape, v.shape[-1]
```

Flake8 (7.3.0)

[error] local variable 'B' is assigned to but never used

(F841)